### ISSUE TITLE
Dust consolidation

### ISSUE THREAD

**Issue by Xariyl** [AlexTheGre4t/MCTNGIssues#36]
On the **To Do** list. 

There are now several types of metal dusts, (char)coal dusts, sulfurs. All are in oreDict, so I want to consolidate them to use a single standard like we do with the metals.

Preferred type? I was thinking TE (pulverized) versions just to be consistent with the metals.

**Comment by AlexTheGre4t** [comment]
Yes, I would prefer them to be all TF just because of consistency. Plus they're the only material types that don't seem to have some wierd prioritisation in recipes :P

Thanks for doing these :D

**Comment by Xariyl** [comment]
Is there a way to make RailCraft's sulfur ore blocks drop Thermal Foundation's type of sulfur when mined? Not sure how to resolve that one.

**Comment by AlexTheGre4t** [comment]
A little difficult, but not impossible, there will be something out there. If not, I may be able to ask for a favour from someone to mod it in :D ill investigate.
